core/src/main/kotlin/io/github/thanosfisherman/gasket/FirstScreen.kt

```diff
@@ -57,18 +68,32 @@ class FirstScreen(private val game: Game) : KtxScreen {
         if (Gdx.input.isKeyPressed(Keys.ESCAPE))
             Gdx.app.exit()
 
-        gasket.draw()
-        fps.render()
+        Gdx.gl.glLineWidth(1.64f)
+
+        gameViewport.apply()
+        shape.projectionMatrix = gameViewport.camera.combined
+
+        shape.begin(ShapeRenderer.ShapeType.Line)
```

Comment:
I like this refactoring. However I don't quite understand how there are less draw calls here since we still call begin/end and draw the same number of circles in the middle.